src/telemetry.ts

```diff
@@ -3174,28 +3190,100 @@ export class IEventNamePropertyMapping {
             }
         }
     };
-    [DebuggingTelemetry.clickedOnSetup]: TelemetryEventInfo<never | undefined> = { owner: 'roblourens' } as any;
-    [DebuggingTelemetry.closedModal]: TelemetryEventInfo<never | undefined> = { owner: 'roblourens' } as any;
+    /**
+     * Sent when the user accepts the prompt to install ipykernel 6 automatically.
+     */
+    [DebuggingTelemetry.clickedOnSetup]: TelemetryEventInfo<never | undefined> = {
+        owner: 'roblourens',
+        feature: ['Debugger'],
+        source: 'User Action'
+    };
+    /**
+     * Sent when the user dismisses the prompt to install ipykernel 6 automatically.
+     */
+    [DebuggingTelemetry.closedModal]: TelemetryEventInfo<never | undefined> = {
+        owner: 'roblourens',
+        feature: ['Debugger'],
+        source: 'User Action'
+    };
+    /**
+     * An event describing whether the environment has ipykernel 6 installed.
+     */
     [DebuggingTelemetry.ipykernel6Status]: TelemetryEventInfo<{
         status: 'installed' | 'notInstalled';
-    }> = { owner: 'roblourens' } as any;
-    [DebuggingTelemetry.clickedRunByLine]: TelemetryEventInfo<never | undefined> = { owner: 'roblourens' } as any;
-    [DebuggingTelemetry.successfullyStartedRunByLine]: TelemetryEventInfo<never | undefined> = {
-        owner: 'roblourens'
-    } as any;
+    }> = {
+        owner: 'roblourens',
+        feature: ['Debugger'],
+        properties: {
+            /**
+             * Whether ipykernel 6 is installed.
```

Comment:
You need to add the jsDoc comments to the property declaration itself.

Reply:
See here in the failure for `npm run generateTelemetry` its on the CI job
```
[19:06:36] Using gulpfile ~/work/vscode-jupyter/vscode-jupyter/gulpfile.js
[19:06:36] Starting 'validateTelemetry'...
Unknown type 153 in generating the Telemetry Documentation
Unknown type 153 in generating the Telemetry Documentation
Problems for roblourens:
    DATASCIENCE.DEBUGGING.ENDED_SESSION (DebuggingTelemetry.endedSession):
        1. Add jsDoc comments to describe this property = reason.
    DATASCIENCE.DEBUGGING.IPYKERNEL6_STATUS (DebuggingTelemetry.ipykernel6Status):
        1. Add jsDoc comments to describe this property = status.
    DATASCIENCE.INTERACTIVE_WINDOW_DEBUG_SETUP_CODE_FAILURE (Telemetry.InteractiveWindowDebugSetupCodeFailure):
        1. Add jsDoc comments to describe this property = ename.
        2. Add jsDoc comments to describe this property = evalue.
Problems for unknown:
    DS_INTERNAL.SETTINGS (Telemetry.DataScienceSettings):
        1. Source not defined
        2. Add jsDoc comments to describe this event.
        3. No owner specified for this event
src/gdpr.ts is not valid, please re-run `npm run generateTelemetry`
```

Reply:
Please run `npm run generateTlemetry` on your machine and commit the changes to the TELEMETRY.md & TELEMETRY.csv and gdpr.ts files.
this is manual and only required when making changes to telemetry.